dask/tests/test_base.py

```diff
@@ -139,6 +139,16 @@ def test_tokenize_numpy_memmap():
 
     assert y != z
 
+    #Test two different memmaps into the same numpy file
+    file = open('demo_data', 'wb')
```

Comment:
This test is failing on windows, and isn't 100% correct on linux either. When testing it's important to isolate temporary files/directories from the local workspace, and ensure they're cleaned up outside the test function.

To do this, it's best practice to use py.test fixtures for creating and managing temporary files (https://docs.pytest.org/en/latest/tmpdir.html). For historical reasons this test function uses `with tmpfile` instead, which is also correct if not ideal.

I'd split your code into a new test, and update the code as follows:

```python
@pytest.mark.skipif('not np')
def test_tokenize_numpy_memmap_offset(tmpdir):
    #Test two different memmaps into the same numpy file
    fn = str(tmpdir.join("demo_data"))

    with open(fn, 'wb') as f:
        f.write(b'ashekwicht')

    with open(fn, 'rb') as f:
        mmap1 = np.memmap(f, dtype=np.uint8, mode='r', offset=0, shape=5)
        mmap2 = np.memmap(f, dtype=np.uint8, mode='r', offset=5, shape=5)

        assert(tokenize(mmap1) != tokenize(mmap2))
```